src/components/media-player/media-hooks.js

```diff
@@ -0,0 +1,27 @@
+import deepEqual from 'fast-deep-equal';
+import { useEffect, useState } from 'react';
+import { useService } from '../container-context.js';
+import MediaDurationCache from './media-duration-cache.js';
+
+export function useMediaDuration(urlOrUrls) {
```

Comment:
Would it be ok for you if we always worked with arrays here?
I'm not a big fan of this 2-in-1 function signature pattern 😬

Reply:
Sure, I'll change it :+1:

Reply:
Done!